client/app/components/packages/scope-of-work-draft/show.hbs

```diff
@@ -1,71 +1,139 @@
-<div class="grid-x grid-margin-x">
-  <div class="cell large-8">
-    <section class="form-section">
-      <h1 class="header-large">
-        Draft Scope of Work (DSOW) Information Confirmation
-        <small
-          class="text-weight-normal"
-          data-test-package-dcpPackageversion
+<SaveableForm 
+  @model={{@package}}
+  @validators={{array (hash) (hash)}} 
+  as |saveableForm|
+>
+
+  <div class="grid-x grid-margin-x">
+    <div class="cell large-8">
+      <section class="form-section">
+        <h1 class="header-large">
+          Draft Scope of Work (DSOW) Information Confirmation
+          <small
+            class="text-weight-normal"
+            data-test-package-dcpPackageversion
+          >
+            {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
+          </small>
+        </h1>
+
+        <h2 class="no-margin"
+          data-test-project-dcpProjectname
         >
-          {{if @package.dcpPackageversion (concat '(V' @package.dcpPackageversion ')')}}
-        </small>
-      </h1>
+          {{@package.project.dcpProjectname}}
+          <small
+            class="text-weight-normal"
+            data-test-project-dcpName
+          >
+            {{if @package.project.dcpName (concat '(' @package.project.dcpName ')')}}
+          </small>
+        </h2>
 
-      <h2 class="no-margin"
-        data-test-project-dcpProjectname
-      >
-        {{@package.project.dcpProjectname}}
-        <small
-          class="text-weight-normal"
-          data-test-project-dcpName
+        <p
+          class="text-large text-dark-gray"
+          data-test-project-dcpBorough
+          data-test-package-statuscode
         >
-          {{if @package.project.dcpName (concat '(' @package.project.dcpName ')')}}
-        </small>
-      </h2>
+          {{optionset 'bbl' 'boroughs' 'label' @package.project.dcpBorough}} |
+          {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
+        </p>
+      </section>
 
-      <p
-        class="text-large text-dark-gray"
-        data-test-project-dcpBorough
-        data-test-package-statuscode
-      >
-        {{optionset 'bbl' 'boroughs' 'label' @package.project.dcpBorough}} |
-        {{optionset 'package' 'statuscode' 'label' @package.statuscode}}
-      </p>
-    </section>
+      {{#if (gte saveableForm.data.ceqrInvoiceQuestionnaires.length 1)}}
+          {{#let saveableForm.data.singleCeqrInvoiceQuestionnaire as |ceqrInvoiceQuestionnaire|}}
+            <saveableForm.Section
+            @title="Ceqr Invoice Questionnaire" 
+            @elementId="ceqr-invoice-questionnaire"
+            >
+              <saveableForm.SaveableForm
+                @model={{ceqrInvoiceQuestionnaire}}
+                @validators={{array (hash) (hash)}} 
+              >
```

Comment:
Why is this necessary? It doesn't look like it is being used. There is no yield like the first SaveableForm above. I.e. no `as |cequirInvoiceSaveableForm|`

I think this was only necessary on the Edit page, but doesn't look necessary here

Reply:
To bring in ceqrInvoiceQuestionnaire. Maybe you have a better way.

Reply:
It should still work without this. 
When you display a value like on line 57, it is referencing the  `ceqrInvoiceQuestionnaire` block variable from line 43. 

Previously on the Edit page, there is a new `saveableForm.SaveableForm` component in order to render Input fields, etc, but now that's not necessary because the user already inputted the data

Reply:
Thanks @godfreyyeung , corrected and also added a test case.